test.sh

```diff
@@ -8,7 +9,7 @@ cargo fmt
 
 cargo clippy
 ( cd test && cargo clippy )
-( cd crates/guest && cargo clippy )
+( cd crates/guest && cargo clippy --target wasm32-unknown-unknown )
```

Comment:
Gets rid of a warning about FFI types, we care about pointer sizes passing linting only on the wasm32 target